### ISSUE TITLE
EBAS variable request

### ISSUE THREAD

**Issue by annefou** [NordicESMhub/NEGI-Abisko-2019#29]
@jgliss will help you...

**Comment by dominichr1** [comment]
aerosol_light_scattering_coefficient for  pm10_humidified and aerosol_humidified

**Comment by dominichr1** [comment]
or could we get the 'pydap' module installed

**Comment by ksenia-tabakova** [comment]
Hi!
1. Zeppelin

Matrix: met
Instrument: aws
Variables: relative_humidity, temperature, wind_direction, wind_speed

Matrix: aerosol:
Variables:cloud_condensation_nuclei_number_concentration,  chlorine, sodium, particle_number_size_distribution, particle_number_concentration, 

2. Ny-Alesund- Nordpolhotellet

Matrix: aerosol
Variables: chlorine, sodium,

**Comment by jgliss** [comment]
> aerosol_light_scattering_coefficient for pm10_humidified and aerosol_humidified

@dominichr1 I need to check how the humidified scattering data is stored in the EBAS NASA Ames files. I imagine there is several columns with different RH in one file (the current pyaerocom reading of `scatc550aer` and `scatc550dryaer` only considers EBAS matrix `aerosol` and `pm10`. 

We can look at this together this afternoon based on one example file and see what is the best reading strategy.

**Comment by jgliss** [comment]
> Hi!
> 
>     1. Zeppelin
> 
> 
> Matrix: met
> Instrument: aws
> Variables: relative_humidity, temperature, wind_direction, wind_speed
> 
> Matrix: aerosol:
> Variables:cloud_condensation_nuclei_number_concentration, chlorine, sodium, particle_number_size_distribution, particle_number_concentration,
> 
>     1. Ny-Alesund- Nordpolhotellet
> 
> 
> Matrix: aerosol
> Variables: chlorine, sodium,

@ksenia-tabakova I will check that, I also suggest we sit together and check what is the best way to read these data into 2 stations. I imaging the met variables should be straight forward, since it is single value variables (e.g. wind speed at 12am). Not sure though, how EBAS stores size distributions, but I imagine that they have multiple data columns in the file, each one corresponding to a certain size range....

**Comment by dominichr1** [comment]
> > aerosol_light_scattering_coefficient for pm10_humidified and aerosol_humidified
> 
> @dominichr1 I need to check how the humidified scattering data is stored in the EBAS NASA Ames files. I imagine there is several columns with different RH in one file (the current pyaerocom reading of `scatc550aer` and `scatc550dryaer` only considers EBAS matrix `aerosol` and `pm10`.
> 
> We can look at this together this afternoon based on one example file and see what is the best reading strategy.

Yes, it's scattering coefficients at different RHs and wavelengths. but, we'll find you this afternoon. We're group 4.

**Comment by marikoll** [comment]
Hello, could I get the data for: 

Zeppelin: 

Matrix: air
Intstument type: filter_3pack
Component: sulphur_dioxide

Ny-Ålesund - Gruvebadet: 

Matrix: air
Intstument type: filter_3pack
Component: sulphur_dioxide

Villum Research station: 

Matrix: air
Intstument type: filter_3pack
Component: sulphur_dioxide

Thanks!

**Comment by jgliss** [comment]
Hi all, 

I added two examples that illustrate more low-level reading of EBAS data, which should enable you to load all that you need for your analysis:

- [How to find the EBAS data files you need](https://github.com/NordicESMhub/NEGI-Abisko-2019/blob/master/content/training/pyaerocom/HOWTO_find_EBAS_files.ipynb)
- [How to read them and extract metadata and timeseries](https://github.com/NordicESMhub/NEGI-Abisko-2019/blob/master/content/training/pyaerocom/HOWTO_read_EBAS_files.ipynb)

**NOTE**: The EBAS dataset we provide under `pyaerocom-testdata/obsdata/EBASMultiColumn` **does not contain everything that is available in the EBAS database**, so if [finding files](https://github.com/NordicESMhub/NEGI-Abisko-2019/blob/master/content/training/pyaerocom/HOWTO_find_EBAS_files.ipynb) fails, you need to download the data files (NASA Ames, file ending is *.nas*) from the [EBAS database](http://ebas.nilu.no/) and then read them as explained [here](https://github.com/NordicESMhub/NEGI-Abisko-2019/blob/master/content/training/pyaerocom/HOWTO_read_EBAS_files.ipynb).

**Comment by olga-firus** [comment]
@ksenia-tabakova 
Here is an example how to get ebas data with pydap package
[content/training/example_notebooks/from_ebas_using_pydap.ipynb](url)
